### DISCUSSION TITLE
Automatically enable org mode in org related files

### DISCUSSION THREAD

**Discussion by Niiisse** [syl20bnr/spacemacs#15407] [Q&A]
Hi all,

How would I go about enabling org mode automatically? I save all org files with .org as extension, but they open in fundamental mode. Having to switch to org mode manually gets a bit cumbersome after s while. Any ideas?

**Comment by Niiisse** [comment]
Figured it out (surprised it didn't come up earlier in my Googling)
`DOC TITLE -*- mode: org -*-` as the first line works like a charm.

**Comment by arifer612** [comment]
It is odd that your Org files are not starting up in Org mode. Can you do a quick check to see what your `auto-mode-alist` is? You can do that by `C-h v auto-mode-alist` or `M-x describe-variable auto-mode-alist`. It should be present by default. But if you don't see it, you can add the following to your `user-config`:

```
    (add-to-list 'auto-mode-alist '("\\.org\\'" . org-mode))
```
